pygamelib/gfx/ui.py

```diff
@@ -31,7 +31,7 @@
 from typing import Union, Optional, Set, List, TYPE_CHECKING
 from pygamelib.assets import graphics
 from pygamelib.gfx import core
-from pygamelib import base, constants
+from pygamelib import base, constants, engine
```

Comment:
you can't import engine here, it will create a loop (and it's not desirable). 
In the latest version of that class engine is dynamically imported in case it's needed.

Reply:
Okay, I removed the engine import, and pushed changes to my original branch 'Add-gfx.ui-type-hints' on PR #244, so #255 can be closed, so sorry about that!